graphql/types/SponsoredProduct.graphql

```diff
@@ -0,0 +1,74 @@
+type SponsoredProduct {
+  """
+  Unique identifier of the product. Can be SKU, EAN or VTEX Product ID.
+  """
+  identifier: ProductUniqueIdentifier
+
+  """
+  ID of the sponsored product.
+  """
+  productId: ID @deprecated(reason: "Use `identifier` instead.")
+
+  """
+  Merchandising rule applied to the product.
+  """
+  rule: Rule
+
+  """
+  Sponsored campaign related information should be added here.
+  """
+  advertisement: Advertisement
+}
+
+type ProductUniqueIdentifier {
+  """
+  Product ID value. If null, consider SKU as the default.
+  """
+  field: ProductUniqueIdentifierField
```

Comment:
Ideally, this field should have a default value as "sku", since relevanC will not implement this.

However, GraphQL does not have a `@default` directive by default.

So, for now, the `intelligent-search-api` app will consider `sku` as default and, later, I'll get back to this schema and implement a `default` directive myself.